suites/message/create_actor.go

```diff
@@ -109,15 +109,15 @@ func TestInitActorSequentialIDAddressCreate(t *testing.T, factory state.Factorie
 	var initialBal = abi_spec.NewTokenAmount(200_000_000_000)
 	var toSend = abi_spec.NewTokenAmount(10_000)
 
-	sender, senderID := td.NewAccountActor(drivers.SECP, initialBal)
+	sender, _ := td.NewAccountActor(drivers.SECP, initialBal)
 
 	receiver, receiverID := td.NewAccountActor(drivers.SECP, initialBal)
 
 	firstPaychAddr := utils.NewIDAddr(t, utils.IdFromAddress(receiverID)+1)
 	secondPaychAddr := utils.NewIDAddr(t, utils.IdFromAddress(receiverID)+2)
 
-	firstInitRet := td.ComputeInitActorExecReturn(senderID, 0, firstPaychAddr)
-	secondInitRet := td.ComputeInitActorExecReturn(senderID, 1, secondPaychAddr)
+	firstInitRet := td.ComputeInitActorExecReturn(sender, 0, 0, firstPaychAddr)
+	secondInitRet := td.ComputeInitActorExecReturn(sender, 1, 0, secondPaychAddr)
```

Comment:
I believe the `senderID` address should be passed here since `ApplyMessage` will [resolve the sender](https://filecoin-project.github.io/specs/#vminterpreter-implementation) address through the InitActor's map before performing any operations.

Currently go-filecoin performs the operations as described in the spec here: https://github.com/filecoin-project/go-filecoin/blob/master/internal/pkg/vm/internal/vmcontext/vmcontext.go#L360

From a quick skim of the lotus code I don't believe this has been instrumented yet: https://github.com/filecoin-project/lotus/blob/testnet/3/chain/vm/vm.go#L252